common/types/firestore-types.ts

```diff
@@ -6,6 +6,11 @@ export type FirestoreCommon = {
   readonly order: number;
 };
 
+export type FirestoreCommonNew = {
```

Comment:
This type is currently unused. The goal is to eventually replace the current type with this after the cron job runs.